indico/web/client/js/react/components/ICSCalendarLink.module.scss

```
@@ -13,10 +13,15 @@
 
 .export-header:global(.ui.header) {
   display: flex;
-  justify-content: center;
+  flex-direction: column;
+  align-items: center;
   margin-bottom: 0;
 }
 
+.export-header:global(.ui.header:first-child) {
```

```suggestion
.export-header:global(.ui.header):first-child {
```

IMO it's more logical to only keep the stuff that actually needs global scoping in there. (untested btw, but I don't see how it would make a difference in the resulting CSS)